theme/apps/gdcmvs/dist/bundle.js

```diff
@@ -152,20 +152,205 @@ var func = {
+/* harmony import */ var __WEBPACK_IMPORTED_MODULE_1__props_table___ = __webpack_require__(8);
+/* harmony import */ var __WEBPACK_IMPORTED_MODULE_2__values_table___ = __webpack_require__(10);
+/* harmony import */ var __WEBPACK_IMPORTED_MODULE_3__tabs___ = __webpack_require__(12);
+/* harmony import */ var __WEBPACK_IMPORTED_MODULE_4__shared__ = __webpack_require__(1);
+
+
+
+
+
+
+function render(keyword, option, items) {
+  var html = "";
+  if (items.length !== 0) {
+    var trsHtml = __WEBPACK_IMPORTED_MODULE_0__result_table___["a" /* default */].render(items);
+    trsHtml.active = false;
+    var psHtml = __WEBPACK_IMPORTED_MODULE_1__props_table___["a" /* default */].render(items);
+    psHtml.active = false;
+    var vsHtml = __WEBPACK_IMPORTED_MODULE_2__values_table___["a" /* default */].render(items);
+    vsHtml.active = false;
+    if (option.activeTab == 0) {
+      vsHtml.active = true;
+    } else if (option.activeTab == 1) {
+      psHtml.active = true;
+    } else {
+      trsHtml.active = true;
+    }
+    html = Object(__WEBPACK_IMPORTED_MODULE_3__tabs___["a" /* default */])(trsHtml, psHtml, vsHtml, keyword);
+  } else if (option.error == true) {
+    html = '<div class="indicator indicator--has-error">Please, enter a valid keyboard!</div>';
+  } else {
+    html = '<div class="indicator">Sorry, no results found for kerword: <span class="indicator__term">' + keyword + '</span></div>';
+  }
+
+  $("#root").html(html);
+
+  if ($("#tree_table").length) {
+    $("#tree_table").treetable({ expandable: true });
+    $("#tree_toggle").bind('click', function () {
+      var target = $(this);
+      if (target.attr("aria-pressed") == 'true') {
+        target.html('<i class="fa fa-angle-down"></i> Expand All');
+        $('#gdc-loading-icon').fadeIn(100);
+
+        setTimeout(function () {
+          $("#tree_table").find('a[title="Collapse"]').each(function () {
+            $(this).trigger("click");
+          });
+          $('#gdc-loading-icon').fadeOut('fast');
+        }, 1000);
+      } else {
+        target.html('<i class="fa fa-angle-up"></i>  Collapse All');
+        $('#gdc-loading-icon').fadeIn(100);
+
+        setTimeout(function () {
+          $("#tree_table").find('a[title="Expand"]').each(function () {
+            $(this).trigger("click");
+          });
+          $("#tree_table").find('a[title="Expand"]').each(function () {
+            $(this).trigger("click");
+          });
+          $("#tree_table").find('a[title="Expand"]').each(function () {
+            $(this).trigger("click");
+          });
+          $('#gdc-loading-icon').fadeOut('fast');
+        }, 1000);
+      }
+    });
+  }
+
+  $('a.redirect').bind('click', function (event) {
+    event.preventDefault();
+    if (window.location.href.indexOf('https://docs.gdc.cancer.gov/') < 0) {
+      var href = $(this).attr('href');
+      window.open('https://docs.gdc.cancer.gov' + href, '_blank');
```

Comment:
when the app is in other domain different to `https://docs.gdc.cancer.gov`, it will a open in a new tap

Reply:
I think the `event.preventDefault();` needs to be inside the `if` otherwise the link doesn't work at all if at same domain

Reply:
thank you @cy